samples/bluetooth/iso_time_sync/src/cis_central.c

```diff
@@ -183,7 +183,13 @@ void cis_central_start(bool do_tx, uint8_t retransmission_number, uint16_t max_t
 		 */
 		iso_tx_init(retransmission_number, &scan_start);
 	} else {
-		iso_rx_init(retransmission_number);
+		/** scan_start is registered as a callback that is triggered when an
+		 * ISO channel disconnects. This is needed because the ISO channel might
+		 * disconnect after the ACL has completed its disconnection callback. If
+		 * this is the case, there are no RX ISO channels available during the
+		 * ACL disconnection callback, which will prevent scanning from starting.
+		 */
+		iso_rx_init(retransmission_number, &scan_start);
```

Comment:
I guess this has the side-effect that we sometimes would print https://github.com/nrfconnect/sdk-nrf/blob/c50248c3788c74bbf4fc4c0c349bde9fdd2c2a80/samples/bluetooth/iso_time_sync/src/cis_central.c#L104C12-L104C45 , if ISO is disconnected before the ACL, right?

Reply:
I think you are right. I can add a check if the error returns that the scanner is already enabled.